region/crateria/east/East Ocean.json

```diff
@@ -450,7 +449,112 @@
     },
     {
       "link": [2, 2],
-      "name": "Leave With Temporary Blue",
+      "name": "Leave With Temporary Blue (Spring Ball Jump)",
+      "requires": [
+        "Gravity",
+        {"and": [
+          {"canShineCharge": {
+            "usedTiles": 15,
+            "openEnd": 0,
+            "steepUpTiles": 2,
+            "steepDownTiles": 3,
+            "startingDownTiles": 1
+          }},
+          "HiJump",
+          "canSpringBallJumpMidAir"
+        ]},
+        "canChainTemporaryBlue"
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      }
+    },
+    {
+      "link": [2, 2],
+      "name": "Leave With Temporary Blue (Gravity Jump)",
+      "requires": [
+        "Gravity",
+        "HiJump",
+        {"or": [
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 21,
+              "openEnd": 0,
+              "steepUpTiles": 3,
+              "steepDownTiles": 3,
+              "startingDownTiles": 1
+            }},
+            "canXRayTurnaround"  
+          ]},
+          {"canShineCharge": {
+            "usedTiles": 18,
+            "openEnd": 0,
+            "steepUpTiles": 2,
+            "steepDownTiles": 3,
+            "startingDownTiles": 1
+          }}
+        ]},
+        "canGravityJump",
+        "canChainTemporaryBlue"
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      }
+    },
+    {
+      "link": [2, 2],
+      "name": "Leave With Temporary Blue (Spring Ball Jump)",
```

Comment:
Looks like there are duplicate strats for Springballjump and Xrayturnaround into springballjump